tests/pdbackup/backup_test.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pdbackup
+
+import (
+	"context"
+	"strings"
+	"testing"
+	"time"
+
+	. "github.com/pingcap/check"
+	"github.com/pingcap/pd/server"
+	"github.com/pingcap/pd/tests"
+	"github.com/pingcap/pd/tools/pd-backup/pdbackup"
+	"go.etcd.io/etcd/clientv3"
+)
+
+func Test(t *testing.T) {
+	TestingT(t)
+}
+
+var _ = Suite(&backupTestSuite{})
+
+type backupTestSuite struct{}
+
+func (s *backupTestSuite) SetUpSuite(c *C) {
+	server.EnableZap = true
+}
+
+func (s *backupTestSuite) TestBackup(c *C) {
```

Comment:
Good. do we need to add test to restore from backup info?

Reply:
@lhy1024 PTAL,thanks.